dist/war/getstarted/index.html

```diff
@@ -130,7 +130,10 @@ <h4 class="mt-5"><font size="+2">APIs</font> (REST, GraphQL, Javadoc) and <font
         <a id="PythonUrlLink" href="https://github.com/ow2-proactive/proactive-python-client#proactive-scheduler-client" target=”_blank”
            title="Python SDK">Python SDK</a>,
         <a id="javaUrlLink" href="https://doc.activeeon.com/javadoc/latest/org/ow2/proactive/scheduler/rest/SchedulerClient.html" target=”_blank”
-           title="JAVA SDK">JAVA SDK</a>.
+           title="JAVA SDK">JAVA SDK</a>
```

Comment:
please add `,` after title="JAVA SDK">JAVA SDK</a>

Reply:
thanks for spotting this !